### PR TITLE
Create ko_kr.json

### PR DESCRIPTION
**Pull request by mindy15963**

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/ko_kr.json [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+	"block.securitycraft.reinforced_stripped_birch_wood": "강화된 껍질 벗긴 자작나무",
+	"block.securitycraft.reinforced_stripped_jungle_wood": "강화된 껍질 벗긴 정글 나무",
+	"block.securitycraft.reinforced_stripped_acacia_wood": "강화된 껍질 벗긴 아카시아나무",
+	"block.securitycraft.reinforced_stripped_dark_oak_wood": "강화된 껍질 벗긴 짙은 참나무",
+	"block.securitycraft.reinforced_oak_wood": "강화된 참나무",
+	"block.securitycraft.reinforced_spruce_wood": "강화된 가문비나무",
+	"block.securitycraft.reinforced_birch_wood": "강화된 자작나무",
+	"block.securitycraft.reinforced_jungle_wood": "강화된 정글 나무",
+	"block.securitycraft.reinforced_acacia_wood": "강화된 아카시아나무",
+	"block.securitycraft.reinforced_dark_oak_wood": "강화된 짙은 참나무,
```
Comment (Geforce132) [inline src/main/resources/assets/securitycraft/lang/ko_kr.json]: There's a missing closing quotation mark at the end of this line.

Comment (mindy15963) [reply src/main/resources/assets/securitycraft/lang/ko_kr.json]: I fixed.

File: src/main/resources/assets/securitycraft/lang/ko_kr.json [hunk 2]
```diff
@@ -0,0 +1,809 @@
...
+
+	"gui.securitycraft:scManual.disabled": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv1": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv2": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv3": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv4": "사용안함",
+	"gui.securitycraft:scManual.recipe.keycard_lv5": "사용안함",
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "사용안함",
+	"gui.securitycraft:scManual.intro.1": "세큐리티크래프트 시작하기",
+	"gui.securitycraft:scManual.intro.2": "Geforce, bl4ckscor3",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/ko_kr.json]: Translate and add
`    "gui.securitycraft:scManual.author": "Translated by mindy15963",`
after this line

**Comment by bl4ckscor3** [comment]
From what I've seen in Google Translate (can't really check any other way), this seems fine. I noted one last thing to do before the merge.

**Pull request merged**